### PR TITLE
MongoDB,Inc.MongoDBAtlasCLI version 1.5.1

### PR DESCRIPTION
**Pull request by gssbzn**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97479)

### CODE REVIEW

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml
Comment (mdanish-kh) [inline manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: MongoDB.MongoDBAtlasCLI
```
There already exists a PackageID and directory for MongoDB packages to be published in. Let's not duplicate it.

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.yaml
Comment (mdanish-kh) [inline manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.yaml:4]:
```suggestion
PackageIdentifier: MongoDB.MongoDBAtlasCLI
```
There already exists a PackageID and directory for MongoDB packages to be published in. Let's not duplicate it.

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml
Comment (mdanish-kh) [inline manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml:9]:
```suggestion
License: Apache-2.0
LicenseUrl: https://raw.githubusercontent.com/mongodb/mongodb-atlas-cli/master/LICENSE
Copyright: Copyright (c) 2023 MongoDB Inc.
CopyrightUrl: https://www.mongodb.com/customer-agreement
```

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml
Comment (mdanish-kh) [inline manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml:7]:
```suggestion
Publisher: MongoDB, Inc.
PublisherUrl: https://www.mongodb.com/
PublisherSupportUrl: https://github.com/mongodb/mongodb-atlas-cli/issues
PrivacyUrl: https://www.mongodb.com/legal/privacy-policy
```
Additional metadata

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml
Comment (mdanish-kh) [inline manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml:8]:
```suggestion
PackageName: MongoDB Atlas CLI
PackageUrl: https://www.mongodb.com/docs/atlas/cli/stable/
```
Additional metadata

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml
Comment (mdanish-kh) [inline manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml:10]:
```suggestion
ShortDescription: The Atlas CLI (mongodb-atlas-cli) lets you manage your MongoDB Atlas deployments from the command line.
Moniker: mongodb-atlas-cli
Tags:
- atlas
- cli
- cloud
- cloud-management
- database
- mongodb
ReleaseNotesUrl: https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-changelog/
Documentations:
- DocumentLabel: Command Reference
  DocumentUrl: https://www.mongodb.com/docs/atlas/cli/stable/command/atlas/
- DocumentLabel: Tutorials
  DocumentUrl: https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-tutorials/
```
Additional metadata

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.installer.yaml
Comment (stephengillie) [inline manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.installer.yaml:4]:
```suggestion
PackageIdentifier: MongoDB.AtlasCLI
```

Comment (stephengillie) [reply manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.installer.yaml:4]: This change will need to be accompanied by updating the file and path names.

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml
Comment (stephengillie) [inline manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: MongoDB.AtlasCLI
```

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.yaml
Comment (stephengillie) [inline manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.yaml:4]:
```suggestion
PackageIdentifier: MongoDB.AtlasCLI
```

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml
Comment (stephengillie) [reply manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml:4]: This change will need to be accompanied by updating the file and path names.

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.yaml
Comment (stephengillie) [reply manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.yaml:4]: This change will need to be accompanied by updating the file and path names.

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml
Comment (stephengillie) [inline manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml:7]:
```suggestion
Publisher: MongoDB
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to modify the directory structure to match the new PackageIdentifier. `manifests/m/MongoDB,Inc/MongoDBAtlasCLI` should be changed to `manifests/m/MongoDB/MongoDBAtlasCLI`. Subsequently, the file names should also be renamed to match the new Package ID.

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required files and make the appropriate changes. 

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml
Comment (stephengillie) [inline manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml:8]:
```suggestion
Publisher: MongoDB
PublisherUrl: https://www.mongodb.com/
PublisherSupportUrl: https://www.mongodb.com/contact
PrivacyUrl:   https://www.mongodb.com/legal/privacy-policy
PackageName: Atlas CLI
PackageUrl: https://www.mongodb.com/tools/atlas-cli
```

**Comment by mdanish-kh** [comment]
Looks like we both were reviewing at the same time 😅

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml
Comment (mdanish-kh) [reply manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml:8]: I believe PackageName and Publisher should match Apps&Features/Control Panel, and the author has put the values accordingly

**Comment by stephengillie** [comment]
> Looks like we both were reviewing at the same time 😅

I've modified my changes to fit with yours, so they should be easier for the author to integrate.

**Comment by gssbzn** [comment]
@stephengillie @mdanish-kh thanks so much for the feedback, I believe I have addressed all of them now

**Review by mdanish-kh — APPROVED** [review]

File: manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml
Comment (mdanish-kh) [reply manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml]: @gssbzn Can you mark this convo as resolved? This is blocking the PR from being merged.
And if you don't have the ability to do that, @stephengillie I think the solution would be to delete this review comment (or manually merge)

Comment (mdanish-kh) [reply manifests/m/MongoDB,Inc/MongoDBAtlasCLI/1.5.1/MongoDB,Inc.MongoDBAtlasCLI.locale.en-US.yaml]: @gssbzn Can you mark this convo as resolved? This is blocking the PR from being merged.
(or @stephengillie can delete the review comment to unblock the automerge)

**Pull request merged**
